.github/workflows/publish-bulk-cdk.yml

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
```

Comment:
You're lucky if you get this new workflow right before it reaches master for the first time 😃

Reply:
I ran it manually and it worked! I was surprised too.